[daikinairbase] Daikin Airbase Binding initial contribution (#5777)
openhab/openhab2-addons: bundles/org.openhab.binding.daikinairbase/README.md

# Daikin Airbase Binding

The Daikin binding allows you to control your Daikin Airbase air conditioning units with openHAB. In order to do so, your Daikin air conditioning unit must have a BRP15B61 WiFi adapter (Also known as Airbase) installed.

## Supported Things

Daikin air conditioning units with a BRP15B61 (also known as an Airbase)installed. 

## Discovery

> This addon will broadcast messages on your local network looking for Daikin Airbase air conditioning units and adding them to the queue of new items discovered. You can also manually add a new item if you know the IP address.

Reviewer: ```suggestion
This binding broadcasts messages on the local network looking for Daikin Airbase air conditioning units and adding them to the Inbox.
```

Author: I believe 7a1a76b50d fixes these comments on README.md
